c/_euphonic.c

```diff
@@ -41,7 +41,7 @@ static PyObject *calculate_phonons(PyObject *self, PyObject *args) {
     // Extra vars only required if dipole = True
     PyArrayObject *py_born;
     PyArrayObject *py_dielectric;
-    double eta;
+    double lambda;
```

Comment:
Lambda is exactly the same as lambda in the paper, so I think for anyone looking at the paper and trying to understand what's happening lambda will be clearer. It was probably a mistake to ever use eta...

This is different from the user input `dipole_parameter` which is a multiplier on lambda. I can amend the relevant commit to explain this so it will be clear to anyone using git blame in the future.